foundry.toml

```
@@ -1,6 +1,10 @@
 [profile.default]
 libs = ['lib']
 test = 'test'
-src = 'proposals/'
-fs_permissions = [{ access = "read", path = "./"}]
+fs_permissions = [{ access = "read", path = "../../"}]
```

add a comment here explaining why do we need this
```suggestion
fs_permissions = [{ access = "read", path = "../../addresses"}]
```

---

we also need read access for `out` folder as well that's why `../../` is being used.